### ISSUE TITLE
Generated Maven pom.xml have incorrect information

### ISSUE THREAD

**Issue by bdemers** [apache/incubator-tuweni#141]
I just updated from `0.10.0` and one of the things I noticed was the pom file changes:

The packaging listed in the pom looks wrong, it's listed as `pom` and it likely should be `jar` (or not defined)
https://repo1.maven.org/maven2/org/apache/tuweni/tuweni-bytes/1.1.0/tuweni-bytes-1.1.0.pom

This is minor, but... the "name" field is wrong in the project poms, this field is commonly used in reports. for example usage of the `maven-notice-plugin` uses the name field.  The "name" `tuweni-bytes` is now `task ':bytes:jar' property 'archiveBaseName'`

Generated output diff example from the `maven-notice-plugin`:
```diff
-  tuweni-bytes under The Apache License, Version 2.0
-  tuweni-crypto under The Apache License, Version 2.0
-  tuweni-io under The Apache License, Version 2.0
-  tuweni-units under The Apache License, Version 2.0
+  task ':bytes:jar' property 'archiveBaseName' under The Apache License, Version 2.0
+  task ':crypto:jar' property 'archiveBaseName' under The Apache License, Version 2.0
+  task ':io:jar' property 'archiveBaseName' under The Apache License, Version 2.0
+  task ':units:jar' property 'archiveBaseName' under The Apache License, Version 2.0
```

NOTE:  I'm just providing the above diff as an example of how the `name` from a pom file _may_ be used.

**Comment by atoulme** [comment]
Indeed, sorry about this. This was fixed in master, I think, though it's hard to test for this without running a local install.

Will be fixed for 1.2.0.

**Comment by atoulme** [comment]
This is now fixed in 1.2.0.
